lang/cy.json

```diff
@@ -1,5 +1,5 @@
 {
-    "cookiesOn": "Cwcis ar Gwneud atwrneiaeth arhosol",
+    "cookiesOn": "Cwcis ar Gwneud atwrneiaeth arhosol'",
```

Comment:
Should this be here? (or should it be curly?)

Reply:
It's a deliberate error to trip the unit test (the issue on the weblate PR)

Approval is to check the branch protection rather than merge